app/code/Magento/Quote/Api/Data/ShippingAssignmentInterface.php

```diff
@@ -34,6 +34,18 @@ public function getItems();
      */
     public function setItems($value);
 
+
+    /**
+     * @param \Magento\Inventory\Model\PackageInterface[]
+     * @return $this
+     */
+    public function setPackages($packages);
+
+    /**
+     * @return \Magento\Inventory\Model\PackageInterface[]
+     */
+    public function getPackages();
```

Comment:
again ShippingAssignment interface has get/setQuoteItems
you also intorduce get/setPackages which also have get/setQuoteItems